designsafe/apps/data/models/taxonomy.py

```diff
@@ -0,0 +1,21 @@
+from datetime import datetime
+from django.db import models
+from django.contrib.auth.models import User
+from django.contrib.auth.validators import ASCIIUsernameValidator
+
+
+
+
+class CustomUser(User):
```

Comment:
Keep it to two spaces before a class definition.
Also, here you are creating a new model based on `User`. This is not needed, if you want to have a user bound to each record in `Vocab` you can do it like so:
```python
user = models.ForeignKey(settings.AUTH_USER_MODEL)
```